actions/src/openstack_check_actions.py

```diff
@@ -0,0 +1,447 @@
+import json
+import os
+from typing import Callable
+import logging
+import sys
+from datetime import datetime
+import requests
+from openstack_api.openstack_connection import OpenstackConnection
+from openstack_action import OpenstackAction
+from requests.auth import HTTPBasicAuth
+
+
+class CheckActions(OpenstackAction):
+
+    # pylint: disable=useless-super-delegation
```

Comment:
This `__init__` block can be deleted all-together. Python will automagically generate this so it's something to potentially go wrong in the future at the moment